### PR TITLE
[#2087] Make project name update on butter ready event

### PR DESCRIPTION
**Pull request by cadecairos**
Lighthouse ticket: https://webmademovies.lighthouseapp.com/projects/65733/tickets/2087

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/fa8de22362ca635/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/bc765bc400e0f0f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/fa8de22362ca635/output.txt

Total script time: 1.60 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/bc765bc400e0f0f/output.txt

Total script time: 1.57 mins

Server is running on http://pr906.butyr.org/templates/basic/
Run the Test Suite: http://pr906.butyr.org/test/

File: src/ui/header.js [hunk 1]
```diff
@@ -186,5 +186,10 @@ define([ "dialog/dialog", "util/lang", "ui/user-data", "ui/widget/tooltip" ],
     butter.listen( "projectsaved", function() {
       _projectName.innerHTML = butter.project.name;
     });
+    butter.listen( "ready", function() {
+      if ( butter.project.name ) {
+        _projectTitle.innerHTML = butter.project.name;
```
Comment (humphd) [inline src/ui/header.js:191]: Do we have any code to escape this title when they first enter it?  What if it has < or > or other HTML characters that will break rendering?  Should we escape it here?

Comment (cadecairos) [inline src/ui/header.js:191]: I'll check that out

Comment (jbuck) [inline src/ui/header.js:191]: We filter this on the server-side when exporting a project, but you're right, we don't filter it when loading a project from `/api/projects/:id`. I think using innerText here would do the trick?

**Pull request merged**